### PR TITLE
Alert component added

### PR DESCRIPTION
**Pull request by humzahasan**

### CODE REVIEW

**Review by iRohitGaur — APPROVED** [review]
Good to merge

**Review by srishtigautam21 — APPROVED** [review]
The class names are understandable....everything looks good you can merge it

File: documentation/alert.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        An alert displays a short, important message in a way that attracts the
+        user's attention without interrupting the user's task.
+      </p>
+      <p class="md-text">Filled Alert</p>
+      <p></p>
+      <div class="filled-alert">
+        <div class="alert alert-danger-filled">
+          <i class="fas fa-exclamation-triangle alert-icon"></i>
+          <p class="sm-text bold-text">
+            Alert on eating Fires without Soft Drink
```
Comment (pradipgharge) [inline documentation/alert.html:30]: Humza here it should be Fries I guess in place of fires😅

Comment (humzahasan) [reply documentation/alert.html:30]: Oh yes. Will push it on the next PR. Thanks @pradipgharge

**Pull request merged**